src/customStyles.css

```diff
@@ -6,6 +6,11 @@
 	margin: 5px 15px  !important;
 }
 
+.active {
+	font-weight: bold;
+	font-size: 0.85rem;
+}
+
```

Comment:
Can you be more specific with this? `.active` is quite generic.
Would something like `#siteHighlight.actice` work?